src/model/cloud-runner/tests/cloud-runner-github-checks.test.ts

```
@@ -17,6 +17,23 @@ describe('Cloud Runner Github Checks', () => {
   if (CloudRunnerOptions.cloudRunnerDebug) {
+    beforeEach(() => {
+      // Mock GitHub API requests to avoid real network calls
+      jest.spyOn(GitHub as any, 'createGitHubCheckRequest').mockResolvedValue({
+        status: 201,
+        data: { id: '1' },
+      });
+      jest.spyOn(GitHub as any, 'updateGitHubCheckRequest').mockResolvedValue({
+        status: 200,
+        data: {},
+      });
+      jest.spyOn(GitHub as any, 'runUpdateAsyncChecksWorkflow').mockResolvedValue(undefined);
```

ESLint

[error] Do not use useless `undefined`.

(unicorn/no-useless-undefined)